web/src/pages/Home.tsx

```diff
@@ -1,10 +1,231 @@
+        />
+        <div className="flex flex-row">
+          <div className=" justify-start">
+            <h2 className="text-center font-bold">Tsujiri</h2>
+            <SponsorsButton />
+          </div>
+          <div className="flex flex-col justify-between content-start mx-10"></div>
+        </div>
+      </div>
+    </>
+  );
+}
+
+function SponsorCard2() {
+  return (
+    <>
+      <div className="bg-white h-80 gap-10 rounded-3xl">
+        <img
+          className="max-w-full overflow-hidden rounded-3xl justify-self-start"
+          src={tsujiri}
+        ></img>
+        <div className="flex flex-row">
+          <div className=" justify-start">
+            <h2 className="text-center font-bold">Tsujiri</h2>
+            <SponsorsButton />
+          </div>
+          <div className="flex flex-col justify-between content-start mx-10"></div>
+        </div>
+      </div>
+    </>
+  );
+}
+
+function SponsorCard3() {
+  return (
+    <>
+      <div className="bg-white h-80 gap-10 rounded-3xl">
+        <img
+          className="max-w-full overflow-hidden rounded-3xl justify-self-start"
+          src={tsujiri}
+        ></img>
+        <div className="flex flex-row">
+          <div className=" justify-start">
+            <h2 className="text-center font-bold">Tsujiri</h2>
+            <SponsorsButton />
+          </div>
+          <div className="flex flex-col justify-between content-start mx-10"></div>
+        </div>
+      </div>
+    </>
+  );
+}
+
+function Landing({ children }: { children: JSX.Element }) {
+  return (
+    <div className="flex flex-col h-screen bg-slate-500 items-center gap-10">
+      {children}
+      <h1 className=" text-center text-3xl font-bold text-gray-900">
+        Home page
       </h1>
+      <img src={uadslogo} className="colour" />
     </div>
   );
 }
+
+function ExampleComponent() {
+  return (
+    <div className="w-screen h-80 bg-orange-500">
+      <h1 className=" text-center text-3xl font-bold text-gray-900">
+        EXAMPLE COMPONENT
+      </h1>
+    </div>
+  );
+}
+
+function SignupSection() {
+  return (
+    <div className="flex flex-col gap-10 h-fit bg-purple-800 space-y-10 p-10">
+      <h1 className="p-10 text-center text-6xl font-bold text-pink-300">
+        Join UADS Today!
+      </h1>
+      <PinkButton linkto="/signup" buttontext="Sign Up Now" />
+    </div>
+  );
+}
+
+function EventSection() {
+  return (
+    <>
+      <div className="flex flex-col gap-10 h-screen bg-yellow-800">
+        <h1 className="p-10 text-center text-3xl font-bold text-pink-300">
+          Our Events
+        </h1>
+        <EventGroups />
+        <PinkButton linkto="/events" buttontext="See More Events" />
+      </div>
+    </>
+  );
+}
+
+function EventGroups() {
+  return (
+    <>
+      <div className="flex flex-row flex-nowrap justify-center gap-5 m-10">
+        <EventCard />
+        <EventCard />
+        <EventCard />
+      </div>
+    </>
+  );
+}
+
+function EventCard() {
```

Comment:
Can use props to pass in information (e.g event title, event date). Lookup online or try looking at the SponsorCard component on how to do this.

Reply:
added props for all relevant elements